dotcom-rendering/src/components/RelativeTime.importable.tsx

```diff
@@ -24,12 +24,14 @@ export const RelativeTime = ({ then, now }: Props) => {
 	useEffect(() => {
 		setDisplay(timeAgo(then));
 		if (!inView) return;
+		const refresh =
+			isString(display) && display.match(/\ds ago/) ? 1_000 : 15_000;
```

Comment:
Would a DOM update every second mean more frequent layout re-calcs, paints and hence main thread blocking which could affect render performance and INP? Particularly so in the first minute as other scripts load and compete for the main thread.

Also it might be distracting to users? It could be something users value but is it worth running this past UX folks?

Reply:
Currently, we are lying to users 14 out of 15 seconds…

Maybe it would be best to only ever update every minute and show “now” for the first minute? Why ever show “xs ago” if we are not going to update more frequently?